src/components/dialogs/parameters/voltageinit/voltage-limits-parameters.js

```diff
@@ -5,90 +5,31 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-import { elementType, useSnackMessage } from '@gridsuite/commons-ui';
-import { Button, DialogActions, IconButton, Tooltip } from '@mui/material';
-import SubmitButton from 'components/dialogs/commons/submitButton';
+import { elementType } from '@gridsuite/commons-ui';
+import { IconButton, Tooltip } from '@mui/material';
 import DndTable from 'components/utils/dnd-table/dnd-table';
 import { EQUIPMENT_TYPES } from 'components/utils/equipment-types';
 import {
     FILTERS,
-    FILTER_ID,
-    FILTER_NAME,
     HIGH_VOLTAGE_LIMIT,
-    ID,
     LOW_VOLTAGE_LIMIT,
-    NAME,
-    PRIORITY,
     SELECTED,
     VOLTAGE_LIMITS,
 } from 'components/utils/field-constants';
-import { useCallback, useEffect, useMemo } from 'react';
-import { FormProvider, useFieldArray, useForm } from 'react-hook-form';
-import { FormattedMessage, useIntl } from 'react-intl';
-import { useSelector } from 'react-redux';
-import {
-    getVoltageInitParameters,
-    updateVoltageInitParameters,
-} from 'utils/rest-api';
+import { useMemo } from 'react';
+import { useFieldArray } from 'react-hook-form';
+import { useIntl } from 'react-intl';
 import InfoIcon from '@mui/icons-material/Info';
 import { VoltageAdornment } from 'components/dialogs/dialogUtils';
-import yup from 'components/utils/yup-config';
-import { yupResolver } from '@hookform/resolvers/yup';
-import { CloseButton, useStyles } from '../parameters';
-
-const formSchema = yup
-    .object()
-    .shape({
-        [VOLTAGE_LIMITS]: yup
-            .array()
-            .of(
-                yup.object().shape({
-                    [FILTERS]: yup
-                        .array()
-                        .of(
-                            yup.object().shape({
-                                [ID]: yup.string().required(),
-                                [NAME]: yup.string().required(),
-                            })
-                        )
-                        .min(1, 'FilterInputMinError'),
-                    [LOW_VOLTAGE_LIMIT]: yup.number().nullable(),
-                    [HIGH_VOLTAGE_LIMIT]: yup.number().nullable(),
-                })
-            )
-            .required(),
-    })
-    .required();
 
-const VoltageLimitsParameters = ({
-    hideParameters,
-    useVoltageInitParameters,
-}) => {
-    const classes = useStyles();
+const VoltageLimitsParameters = ({ control }) => {
```

Comment:
After discussing it with @archendel, `control` is not needed anymore since useFieldArray is declared inside a FormProvider